lib/resistance/game.ex

```diff
@@ -21,6 +21,8 @@ defmodule Game.Server do
   use GenServer
   require Logger
 
+  @quest_config %{1 => 2, 2 => 3, 3 => 2, 4 => 3, 5 => 3}
```

Comment:
Oh hah, that's clever! Didn't think of doing that. I might refactor the max players in PreGame to use the same since we're still waiting on other changes anyway.